src/utils/misc.js

```diff
@@ -32,3 +32,18 @@ export function groupItemsById<T: { +id: number }>(items: T[]): { [id: number]:
 }
 
 export const isValidEmailFormat = (email: string = ''): boolean => /\S+@\S+\.\S+/.test(email);
+
+export const initialsForGroupIcon = (name: string): string => {
```

Comment:
Despite `initialsFromString` using it, `name` isn't a good description of what this function takes as an argument.